drivers/network/tdihelpers/include/tdiconn.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+typedef VOID *PTDI_CONNECTION_INFO_PAIR;
+
+PTRANSPORT_ADDRESS TaCopyTransportAddress( PTRANSPORT_ADDRESS OtherAddress );
+PTRANSPORT_ADDRESS TaBuildNullTransportAddress(UINT AddressType);
+UINT TaLengthOfAddress( PTA_ADDRESS Addr );
+UINT TaLengthOfTransportAddress( PTRANSPORT_ADDRESS Addr );
+VOID TaCopyTransportAddressInPlace( PTRANSPORT_ADDRESS Target,
+				    PTRANSPORT_ADDRESS Source );
+UINT TdiAddressSizeFromType( UINT Type );
+UINT TdiAddressSizeFromName( PTRANSPORT_ADDRESS Name );
+NTSTATUS TdiBuildConnectionInfoInPlace
+( PTDI_CONNECTION_INFORMATION ConnInfo, PTRANSPORT_ADDRESS Name );
+NTSTATUS TdiBuildConnectionInfo
+( PTDI_CONNECTION_INFORMATION *ConnectionInfo, PTRANSPORT_ADDRESS Name );
+NTSTATUS TdiBuildNullConnectionInfoInPlace
+( PTDI_CONNECTION_INFORMATION ConnInfo, ULONG Type );
+NTSTATUS TdiBuildNullConnectionInfo
+( PTDI_CONNECTION_INFORMATION *ConnectionInfo, ULONG Type );
```

Comment:
```suggestion
PTRANSPORT_ADDRESS TaCopyTransportAddress(PTRANSPORT_ADDRESS OtherAddress);
PTRANSPORT_ADDRESS TaBuildNullTransportAddress(UINT AddressType);
UINT TaLengthOfAddress(PTA_ADDRESS Addr);
UINT TaLengthOfTransportAddress(PTRANSPORT_ADDRESS Addr);
VOID TaCopyTransportAddressInPlace(PTRANSPORT_ADDRESS Target, PTRANSPORT_ADDRESS Source);
UINT TdiAddressSizeFromType(UINT Type);
UINT TdiAddressSizeFromName(PTRANSPORT_ADDRESS Name);
NTSTATUS TdiBuildConnectionInfoInPlace(PTDI_CONNECTION_INFORMATION ConnInfo, TRANSPORT_ADDRESS Name);
NTSTATUS TdiBuildConnectionInfo(PTDI_CONNECTION_INFORMATION *ConnectionInfo, TRANSPORT_ADDRESS Name);
NTSTATUS TdiBuildNullConnectionInfoInPlace(PTDI_CONNECTION_INFORMATION ConnInfo, ULONG Type);
NTSTATUS TdiBuildNullConnectionInfo(PTDI_CONNECTION_INFORMATION *ConnectionInfo, ULONG Type);
```
**Mandatoy**: No spaces within the curved braces for func-declarations or calls.

If a line is too long to hold all parameters you are allowed to do the line-breaks like that:
`
NTSTATUS
TdiBuildConnectionInfoInPlace(PTDI_CONNECTION_INFORMATION *ConnectionInfo, ULONG Type);
`
or like that:
`
NTSTATUS TdiBuildConnectionInfoInPlace(
    PTDI_CONNECTION_INFORMATION *ConnectionInfo,
    ULONG Type);
`
or even like that:
`
NTSTATUS TdiBuildConnectionInfoInPlace(PTDI_CONNECTION_INFORMATION *ConnectionInfo,
    ULONG Type);
`
Notice that indentation is just 4 spaces here! The indentation is **not** the opening `(`
because that leads to the need of many follow-up-edits for indent in case a functions name is changed.

Reply:
Hi again indent

     indent -o %2 -gnu -bli0 -hnl -nut -i4 -l110 -bls -ncs -npcs -nprs -nsaf -nsob -lp -cli4 -cbi0 %1

should have fixed that.